tests/subsys/dfu/dfu_target_mcuboot/pm_config.h

```diff
@@ -0,0 +1,128 @@
+/* generated file copied to simplify building the test */
```

Comment:
I suggest removing (or commenting out) all values not needed for the test for the benefit of future readers :).

Reply:
will do!